scripts/executeLocalProtocolOwnershipTransfer.ts

```diff
@@ -20,6 +20,11 @@ import { GnosisBatch } from "./prepareProtocolOwnershipTransfer";
 
 export const ARB_OWNER_PRECOMPILE = "0x0000000000000000000000000000000000000070";
 
+// This is v1.1 imported from the new repo and used by rollup creator
+// TODO: remove the v1.0 upgrade executor in this repo
```

Comment:
Should we address this todo here, or is there another PR for this?

Reply:
let's address it later, it doesn't seems too trivial (and fwiw arb1 is still using v1.0)